nostr.org version 2.0

elidyweaver opened the pull request:
## Design
bullet point format
new images

## New sections
updated information to cater to a wider audience

- You own your nostr
- Nostr gives you choice
- Supporting creators
- Get the Nostr!
- Build on nostr
   - A section to cater to developers
- Have a question about nostr? (footer)
  - Instead of FAQ:  #asknostr

![nostr-org-2024-07-july-1](https://github.com/user-attachments/assets/fbff8571-ff49-4d4e-9040-e2f4c18207ec)
![nostr-org-2024-07-july-2](https://github.com/user-attachments/assets/5cd95e4d-c4aa-4371-b527-8a8fce9e5e40)
![nostr-org-2024-07-july-3](https://github.com/user-attachments/assets/8f67d316-a193-4127-b814-c1c8dc64272e)
![nostr-org-2024-07-july-4](https://github.com/user-attachments/assets/f9008bf3-641e-4280-ba36-42ab9045e539)
![nostr-org-2024-07-july-5](https://github.com/user-attachments/assets/13ef2ee0-33ee-474c-ba1c-38b46d665748)
![nostr-org-2024-07-july-6](https://github.com/user-attachments/assets/9c97a005-341f-49c5-b921-d5eecc5fcd8d)
![nostr-org-2024-07-july-7](https://github.com/user-attachments/assets/6eaf05ed-82a3-4413-bf66-c1c177d134b4)

fiatjaf commented:
I like this a lot more.

elidyweaver commented:
I'd like to add a point on data, because practices on nostr differ greatly from those on social media platforms. 
How would you describe the difference?

elidyweaver commented:
> I like this a lot more.

Grateful.

arcbtc commented:
> I'd like to add a point on data, because practices on nostr differ greatly from those on social media platforms. 
> How would you describe the difference?

This is great. Well done. 

I'd say also leave room for the "other stuff"

On data in social, such as discovery, I'd say it propogates more naturally without tge central planners that exist in traditional platforms.

BenGWeeks commented:
Awesome work. Great design.

flyingisawol commented:
super impressive. love the cranes, the color and the content. Well done

arcbtc commented:
Will have proper review later today. Lgtm! 🚀

elidyweaver commented:
> > I'd like to add a point on data, because practices on nostr differ greatly from those on social media platforms.
> > How would you describe the difference?
> 
> This is great. Well done.
> 
> I'd say also leave room for the "other stuff"

Great point. Gives me a few ideas to highlight the constant nostr development and future possibilities. I'll work on that. 

> 
> On data in social, such as discovery, I'd say it propagates more naturally without the central planners that exist in traditional platforms.

The less centralized planning is a good point. I was thinking more in terms of people not being the "product" for advertisers, and on data not being collected. 

The way nostr works  is a great improvement on data collection in comparison to social media platforms.

I know there are some details like IP and  also metadata on some implementations of DM's, or loss of privacy if your key is lost. Yet more work is being done on privacy in general to address many of these issues.

Is there something else I should be aware of?

elidyweaver commented:
> Will have proper review later today. Lgtm! 🚀

Thanks so much!

elidyweaver commented:
## New updates & improvements
Thanks for everyone's feedback.  

The Other Stuff gets a special section to highlight the importance of constant nostr development without centralized control.

![the-other-stuff](https://github.com/user-attachments/assets/4e04393c-3648-46eb-a679-9d5aa5cf615d)

New bullet point for data

![data](https://github.com/user-attachments/assets/3c377d92-26ff-41c2-9338-4918f40ea4fb)

Added a dropdown on keys to reduce the amount of text for people who already know about them. Added point on AI.

![keys](https://github.com/user-attachments/assets/e1ff83e6-cccf-457c-86f4-291e2f92f6ee)

Added satellite.earth, which had been pending for update based on [Issue #6 ](https://github.com/lnbits/nostr.org/issues/6)

Updated the point on nostr not being a blockchain to also include that it is not a cryptocurrency either

arcbtc commented:
cam you mention people are building marketplaces and IoT on nostr. On dot com we have "A better internet is possible: decentralize Twitter, eBay, IoT and other stuff."

elidyweaver commented:
> How about for the subtitle:
> 
> "the free and open protocol, for building a free and open internet."
> 
> How about for "The Other Stuff":
> 
> "The Other Stuff A better internet is possible! Nostr is not just social media. The paradigm shift in Nostr that empowers users, is being used to create marketplaces, IoT, DNS systems and even entire websites."
> 
> Can you please add:
> 
> * horse (for hw signers) under extensions  https://chromewebstore.google.com/detail/horse/ogdjeglchjlenflecdcoonkngmmipcoe?hl=en-US&pli=1
> * market.nostr.com under webapps
> * YokiHonne under andoid and IoS
> 
> Bugs: Video popup seems broken ![image](https://private-user-images.githubusercontent.com/33088785/353669151-e218c3c8-12cd-4091-993c-1d495e7ed665.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0Mzg4NzMsIm5iZiI6MTcyMjQzODU3MywicGF0aCI6Ii8zMzA4ODc4NS8zNTM2NjkxNTEtZTIxOGMzYzgtMTJjZC00MDkxLTk5M2MtMWQ0OTVlN2VkNjY1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzMxVDE1MDkzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYyNjk1YzZhNTFhZThiYTJlNDZiMjgzOTIwM2VlMDMwYTQyYTgzY2NmZThkYTQ3ZDI5YTU2YzhlOWVjYTZkZjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3diQcvMCKLlVS9m1MyPTtCeEL0v6_rEt8ui7ZBJdSeE)

Thank you, I'll look into all of these and send another update.

elidyweaver commented:
> How about for the subtitle:
> 
> "the free and open protocol, for building a free and open internet."
> 

I'm suggesting a simplified version, hoping it still embodies the concept you're hoping to share.

![free-and-open-internet](https://github.com/user-attachments/assets/4a6a2750-8913-4e3c-abcb-1b32bd9ff848)

> How about for "The Other Stuff":
> 
> "The Other Stuff A better internet is possible! Nostr is not just social media. The paradigm shift in Nostr that empowers users, is being used to create marketplaces, IoT, DNS systems and even entire websites."
> 

As I read your ideas, I thought: What makes nostr a better internet?
I think you answered it in **empowerment**. 
What do you think of this version?

![Screenshot 2024-07-31 at 23 24 09](https://github.com/user-attachments/assets/4e8daab5-c406-46ce-a8c3-2e6d577a7f2c)

> Can you please add:
> 
> * horse (for hw signers) under extensions  https://chromewebstore.google.com/detail/horse/ogdjeglchjlenflecdcoonkngmmipcoe?hl=en-US&pli=1
> * market.nostr.com under webapps
> * YokiHonne under andoid and IoS
> 

Still have to add these. I'll work on that next.

Fixed the video bug. Thanks for noticing. 

> Bugs: Video popup seems broken ![image](https://private-user-images.githubusercontent.com/33088785/353669151-e218c3c8-12cd-4091-993c-1d495e7ed665.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0OTA4MTAsIm5iZiI6MTcyMjQ5MDUxMCwicGF0aCI6Ii8zMzA4ODc4NS8zNTM2NjkxNTEtZTIxOGMzYzgtMTJjZC00MDkxLTk5M2MtMWQ0OTVlN2VkNjY1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODAxVDA1MzUxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY1ZmNjMTFlYTM4ZTI3ZWY3OGJlNmViYzlmMThhN2VlNDM5YmE4OWUwYjk0M2YyZDE2ZDhiOTJjNDkyYjk3MzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Rb2yJdu2VaLtVYwkqbYUUtCu13U6KeicqILDOMsoVOk)

elidyweaver commented:
# Get the Nostr! 
## section update

With the addition of the following clients and extension, the list looked too busy. I've done some design updates.

Highlighter
HiveTalk
Horse
Nostr Market
Nostrnests
YakiHonne (Android, iOS, web)
Zap stream


Desktop version

![desktop](https://github.com/user-attachments/assets/501239bb-1c02-4301-b284-7365b73b27a2)

Mobile version

![mobile](https://github.com/user-attachments/assets/39735a47-854a-4f22-8fa2-54dfd07c0819)

arcbtc commented:
Sorry for being absent, had a holiday. Really want to get this merged soon!
Going to test again. Thank you for all the updates 🙌

arcbtc commented:
On other stuff, how about: "An internet that empowers users is possible."

elidyweaver commented:
> On other stuff, how about: "An internet that empowers users is possible."

Or people instead of users

"An internet that empowers people is possible."

BenGWeeks commented:
What was latest on this? :-)

fiatjaf commented:
This is great, now we just have to remove the ostrich.
